packages/react-storage/src/components/StorageBrowser/Views/LocationActionView/UploadControls.tsx

```diff
@@ -201,13 +201,14 @@ export const UploadControls = (): JSX.Element => {
   const [{ history, path }] = useControl({ type: 'NAVIGATE' });
   const [files, setFiles] = React.useState<File[]>([]);
   const [fileSelect, handleSelect] = useFileSelect(setFiles);
-
+  const [preventOverwrite, setPreventOverwrite] = React.useState<boolean>(true);
```

Comment:
Also no need to pass the generic to `useState` here, the type will be inferred from the default argument

Reply:
Thanks, updated this in latest. I changed the variable to just "overwrite/setOverwrite" also in latest, so that it matches the language of the checkbox.